mapserver/c2cgeoportal.map.mako

```diff
@@ -65,6 +65,7 @@ MAP
             "wms_title" "Démo du c2cgeoportal"
             "wms_abstract" "Des exemples de couches à partir de données OpenData de différentes villes françaises et suisse."
             "wms_onlineresource" "https://${host}/${instanceid}/wsgi/mapserv_proxy"
+            "wfs_onlineresource" "https://${host}/${instanceid}/wsgi/mapserv_proxy"
```

Comment:
you can also use ows_onlineresource to set up both wms and wfs onlineresource.

Reply:
ah, ok. Then that would be better?

Reply:
both are great imho, but mine saves one line (wooh!)